src/app/containers/Ad/Amp/index.test.jsx

```diff
@@ -14,7 +15,7 @@ const adJsonAttributes = slotType => ({
   },
 });
 
-const adWithContext = slotType => (
+const adWithContext = (slotType, enabled = false) => (
```

Comment:
```suggestion
const adWithContext = (slotType, placeholderEnabled = false) => (
```
When I first read this I thought this toggle indicated ads themselves were disabled, just a suggestion to clear that up